src/averageAges.js

```diff
@@ -37,7 +47,17 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const listToCalculate = withChildren
+    ? people.filter(
+      person => person.sex === 'f' && people.some(
+        child => child.mother === person.name))
+    : people.filter(person => person.sex === 'f', 0);
```

Comment:
you don't need second param for filter
```suggestion
    : people.filter(person => person.sex === 'f');
```